packages/backend/src/utils/__tests__/siret.test.js

```diff
@@ -0,0 +1,18 @@
+const { formatSiren, formatSiret } = require("../siret");
+
+describe("Siret", () => {
+  it("Devrait retourner un siret correctement formatté", () => {
```

Comment:
```suggestion
describe("formatSiret", () => {
  it("should format SIRET if it's valid", () => {
```
- pour plus de clareté dans le `descibe` on peut mettre le nom de la fonction.
- pour la description, par convention on utilise le pattern "should ... if/when/for ..."